transform/models/marts/repo_pushes_daily.sql

```diff
@@ -1,9 +1 @@
-SELECT 
-  repo_id, 
-  repo_name, 
-  DATE_TRUNC('day', event_date) AS date_day, 
-  COUNT(*) AS star_count,
-  SUM(COUNT(*)) OVER (PARTITION BY repo_id ORDER BY DATE_TRUNC('day', event_date)) AS cumul_push_count
-FROM {{ ref("stg_gharchive") }} 
-WHERE event_type = 'Push'
-GROUP BY 1, 2, 3
\ No newline at end of file
+{{repo_event_by_period("commit", "day")}}
```

Comment:
Minor style: Let's add spaces at the beginning and end of the curly brackets content `{{ repo... "day") }}`